onboarding/src/Components/EmployeeEdit/EmployeeEditForm.js

```diff
@@ -6,37 +6,45 @@ import ModalWarning from "../ModalWarning";
 
 function EmployeeEditForm(props) {
     document.title = "Onboarding: dodaj pracownika";
-    const fileNameRef = useRef("");
-    const [employeeModal, setModal ] = useState(<></>);
 
     let userCp = {id: 0, name: "", last_name: "", email: "", tel: "",
         position: "", department: "", location: "", avatar: "/onboarding/static/images/unknown-profile.jpg"};
     if(props.user)
         userCp = {...props.user};
 
-    let imageBox = <img src={ userCp.avatar } alt="avatar" />;
-    if(props.enableUploadAvatar){
```

Comment:
Unfortunately this change gives fake functionality for HR to change an avatar for employee,
but it doesn't work because of back-end authentication credentials - logged user can not change others avatar. :confused: 

Property `props.enableUploadAvatar` was to limit this, so that the logged-in can change only his own.